src/waveform/renderers/waveformoverviewrenderer.h

```diff
@@ -8,34 +8,33 @@
 class QPainter;
 class WaveformSignalColors;
 
-class WaveformOverviewRenderer {
-  public:
-    static QImage render(ConstWaveformPointer pWaveform,
-            mixxx::OverviewType type,
-            const WaveformSignalColors& signalColors,
-            bool mono = false);
-    /// These paint methods allow "mono" rendering (mono-mixdown, bottom-aligned).
-    /// Note: Don't use with WOverview, it's not adjusted yet! It does some
-    /// additional scaling for normalization which will atm cut off the bottom part.
-    static void drawWaveformPartRGB(
-            QPainter* pPainter,
-            ConstWaveformPointer pWaveform,
-            int* start,
-            int end,
-            const WaveformSignalColors& signalColors,
-            bool mono = false);
-    static void drawWaveformPartLMH(
-            QPainter* pPainter,
-            ConstWaveformPointer pWaveform,
-            int* start,
-            int end,
-            const WaveformSignalColors& signalColors,
-            bool mono = false);
-    static void drawWaveformPartHSV(
-            QPainter* pPainter,
-            ConstWaveformPointer pWaveform,
-            int* start,
-            int end,
-            const WaveformSignalColors& signalColors,
-            bool mono = false);
-};
+namespace waveformOverviewRenderer {
+QImage render(ConstWaveformPointer pWaveform,
+        mixxx::OverviewType type,
+        const WaveformSignalColors& signalColors,
+        bool mono = false);
+/// These paint methods allow "mono" rendering (mono-mixdown, bottom-aligned).
+/// Note: Don't use with WOverview, it's not adjusted yet! It does some
+/// additional scaling for normalization which will atm cut off the bottom part.
```

Comment:
Is this comment still relevant? It seems the code is used in WOverview

Reply:
Sorry it was not clear enough, should be
"Note: Don't use mono = true with WOverview, it's not adjusted yet!"
Fixed.